### DISCUSSION TITLE
hack-browser-data-0.4.5

### DISCUSSION THREAD

**Discussion by moonD4rk** [moonD4rk/HackBrowserData#263] [Announcements]
## 🚀 Features
* feat: add sogou and dc browser by @BeichenDream in https://github.com/moonD4rk/HackBrowserData/pull/180
* feat: format project layout by @moonD4rk in https://github.com/moonD4rk/HackBrowserData/pull/199
* feat: add full-export browsing data option by @moonD4rk in https://github.com/moonD4rk/HackBrowserData/pull/200
* feat: support Arc browser for macOS by @moonD4rk in https://github.com/moonD4rk/HackBrowserData/pull/220
* feat: improve extension parsing by @stevenlele in https://github.com/moonD4rk/HackBrowserData/pull/235

## 🐛 Bug Fixes
* fix: log error & remove deprecated linter by @carr0t2 in https://github.com/moonD4rk/HackBrowserData/pull/171
* fix: handle error if run binary failed by @moonD4rk in https://github.com/moonD4rk/HackBrowserData/pull/177
* fix: issue of data missing caused by title being null by @zhe6652 in https://github.com/moonD4rk/HackBrowserData/pull/191
* fix: the bug that the command line Usage prompt is incomplete by @slark-yuxj in https://github.com/moonD4rk/HackBrowserData/pull/196
* fix: wrong log caller skip level by @moonD4rk in https://github.com/moonD4rk/HackBrowserData/pull/198
* fix: Bug fixes in firefox file walk by @a-urth in https://github.com/moonD4rk/HackBrowserData/pull/229
* fix: warn if failed to decrypt master key https://github.com/moonD4rk/HackBrowserData/pull/234
* fix: Improve error handling in browsing data and file copying functions by @moonD4rk in https://github.com/moonD4rk/HackBrowserData/pull/261

## ♻️ Chore & Refactor
* refactor: optimize JSON indent output format by @stevenlele in https://github.com/moonD4rk/HackBrowserData/pull/236
* refactor: add provider software folder and check linters by @moonD4rk in https://github.com/moonD4rk/HackBrowserData/pull/172
* chore: upgrade go mod, remove json iterator dep by @moonD4rk in https://github.com/moonD4rk/HackBrowserData/pull/178
* chore: upgrade package dependencies by @moonD4rk in https://github.com/moonD4rk/HackBrowserData/pull/197
* chore: update logo filetype by @moonD4rk in https://github.com/moonD4rk/HackBrowserData/pull/227
* chore: use better bytes comparison method by @testwill in https://github.com/moonD4rk/HackBrowserData/pull/257
* chore: use better bytes comparison method @testwill in https://github.com/moonD4rk/HackBrowserData/pull/257

## New Contributors
* @carr0t2 made their first contribution in https://github.com/moonD4rk/HackBrowserData/pull/171
* @zhe6652 made their first contribution in https://github.com/moonD4rk/HackBrowserData/pull/191
* @slark-yuxj made their first contribution in https://github.com/moonD4rk/HackBrowserData/pull/196
* @a-urth made their first contribution in https://github.com/moonD4rk/HackBrowserData/pull/229
* @BeichenDream made their first contribution in https://github.com/moonD4rk/HackBrowserData/pull/180
* @testwill made their first contribution in https://github.com/moonD4rk/HackBrowserData/pull/257

**Full Changelog**: https://github.com/moonD4rk/HackBrowserData/compare/v0.4.4...v0.4.5

<hr /><em>This discussion was created from the release <a href='https://github.com/moonD4rk/HackBrowserData/releases/tag/v0.4.5'>hack-browser-data-0.4.5</a>.</em>

**Comment by eidson-ciao** [comment]
expect chrome cookie

**Reply by moonD4rk** [reply]
Yep, this is a known issue. You can check the discussion on this https://github.com/moonD4rk/HackBrowserData/issues/243 for details. The current solution is to temporarily not export cookies. This bug will be fixed in future versions.

**Reply by eidson-ciao** [reply]
Thank you very much for the code and ideas for penetration work.

**Comment by James856674** [comment]
Hello, everyone
I wanna decrypt password.
But chrome is updated. So I can't decrypt password.
who can decrypt password?
